Que-Me-Pongo/src/main/java/utils/emailSender.java

```diff
@@ -0,0 +1,108 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Properties;
+import java.util.Set;
+import java.util.StringTokenizer;
+
+import javax.mail.BodyPart;
+import javax.mail.Message;
+import javax.mail.Multipart;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+
+import com.sun.mail.smtp.SMTPTransport;
+
+import modelo.clases.Evento;
+import modelo.clases.Usuario;
+import modelo.interfaces.NotificationManager;
+
+
+public class emailSender implements NotificationManager{
+
+	public emailSender () {}
+	
+	@Override
+	public Properties getProyectProperties() throws Exception {
+		Properties prop = new Properties();
+		InputStream input = null ;
+		try {
+			input = new FileInputStream("system.properties");
+		} catch (FileNotFoundException e) {
+			e.printStackTrace();
+			throw new Exception("No se encuentra el archivo de properties.");
+		}
+
+		try {
+			prop.load(input);
+		} catch (IOException e) {
+			e.printStackTrace();
+			throw new Exception("No se pudo cargar el archivo de properties.");
+		}	
+
+		return prop;
+	}
+	
+	@Override
```

Comment:
This is already on the utils file. Do not override it.

Reply:
Se dejo el método en utils y se saco del emailSender